### ISSUE TITLE
Support alternate compilers.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#46]
```
What steps will reproduce the problem?
1. CXX=CC gmake

What is the expected output? What do you see instead?

CC should be used as the C++ compiler.  g++ is instead used.

What version of the product are you using? On what operating system?

git tip. Solaris.

Please provide any additional information below.


```

---

Original issue reported on code.google.com by `theo.sch...@circonus.com` on 10 Oct 2011 at 9:20

**Comment by GoogleCodeExporter** [comment]
```
This patch fixes the issue and passes CXX param into the make/build system.  
It's small and simple
```

---

Original comment by `theo.sch...@circonus.com` on 10 Oct 2011 at 9:22

Attachments:
- [ldb-alternate-compilers.patch](https://storage.googleapis.com/google-code-attachments/leveldb/issue-46/comment-1/ldb-alternate-compilers.patch)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in c8c5866a86c8. Apologies for the delay.
```

---

Original comment by `h...@chromium.org` on 30 Nov 2011 at 11:09
- Changed title: **Support alternate compilers.**
- Changed state: **Fixed**
